packages/react-api/src/Api.tsx

```diff
@@ -126,7 +127,13 @@ export default function Api ({ children, url }: Props): React.ReactElement<Props
     const provider = new WsProvider(url);
     const signer = new ApiSigner(queuePayload, queueSetTxStatus);
 
-    api = new ApiPromise({ provider, registry, signer, types: { ...CENNZRuntimeTypes as any } });
+    api = new ApiPromise({
```

Comment:
Can we use the CENNZnet api instance for ApiPromise over '@polkadot/api/promise'?

Reply:
With that I feel, we won't require to decorate the RPC calls in UI.. we can use it from API...

Reply:
I tried `cennznet/api/rpc` at the first place, but its not working (need to make a patch release to fix it, and adopt into this PR), so before taking the action in api, I created this PR to get it generally work here. I am okay with either way, what do you think @holygits

Reply:
Recently I am learning [open-closed principle](https://hackernoon.com/why-the-open-closed-principle-is-the-one-you-need-to-know-but-dont-176f7e4416d), and think we may benefit from it in our future projects. @holygits @KarishmaBothara

Reply:
Yeah the RPC definitions should be provided in API, IDM merging this though as it seems simple to fix after a new API release

Reply:
Ok, I will make a PR in api on it now

Reply:
sure..all good !!

Reply:
@holygits @KarishmaBothara , I have created [this PR](https://github.com/cennznet/api.js/pull/102) in api, and could you plz have a look? After that PR merged, we could make a api release, and then I can import the latest api and do the refactoring as required here.